gateway-js/src/config.ts

```diff
@@ -121,69 +126,174 @@ interface GatewayConfigBase {
   experimental_didResolveQueryPlan?: Experimental_DidResolveQueryPlanCallback;
   experimental_didFailComposition?: Experimental_DidFailCompositionCallback;
   experimental_didUpdateComposition?: Experimental_DidUpdateCompositionCallback;
+  /**
+   * @deprecated use `pollIntervalInMs` instead
+   */
   experimental_pollInterval?: number;
+  pollIntervalInMs?: number;
   experimental_approximateQueryPlanStoreMiB?: number;
   experimental_autoFragmentization?: boolean;
   fetcher?: typeof fetch;
   serviceHealthCheck?: boolean;
 }
 
-export interface RemoteGatewayConfig extends GatewayConfigBase {
+// TODO(trevor:removeServiceList)
+export interface ServiceListGatewayConfig extends GatewayConfigBase {
+  /**
+   * @deprecated: use `supergraphSdl` in its function form instead
```

Comment:
These comments could say "use `supergraphSdl: new IntrospectAndCompose(...)` instead" and similarly for all the other ones

Reply:
5158be79